book/website/ethical-research/activism/activism-env-impact.md

```diff
@@ -1,6 +1,16 @@
 (er-activism-env-impact)=
 # The Environmental Impact of Digital Research
 
+```{figure} ../../figures/environmental-impact.*
+---
+height: 400px
+name: environmental-impact
+alt: Cartoon-like sketch depicting the potential environmental impact of digital research. On the left, a person sits at a desk with a laptop, with a chatbot and "COMPUTING" text above, symbolizing digital communication. In the center, "EMISSIONS" emerge from a factory, representing pollution. On the right, a figure throws computers into the ocean, labeled "WASTE," indicating electronic disposal issues.

```

Comment:
```suggestion
alt: Cartoon-like sketch depicting the potential environmental impact of digital research. On the left, a person sits at a desk with a laptop, with a chatbot and "COMPUTING" text above, symbolizing digital communication. In the center, "EMISSIONS" emerge from a factory, representing pollution. On the right, a figure throws computers into the ocean, labeled "WASTE", indicating electronic disposal issues.

```

I would also edit this to add the colour scheme to the alt-text - fixed a minor typo as well 👍🏻